src/react-apps/applications/ux-editor/src/sagas/formDesigner/formDesignerSagas.ts

```diff
@@ -43,30 +42,28 @@ function* addFormComponentSaga({
 }: FormDesignerActions.IAddFormComponentAction): SagaIterator {
   try {
     const id: string = uuid();
-    const selectActiveContainer = (state: IAppState) => state.formDesigner.layout.activeContainer;
-    let activeContainer = yield select(selectActiveContainer);
+
     const formDesignerState: IFormDesignerState = yield select(selectFormDesigner);
 
-    if (containerId) {
-      if (formDesignerState.layout.containers && Object.keys(formDesignerState.layout.containers).length > 0) {
-        activeContainer = containerId;
-      } else {
-        activeContainer = uuid();
-        const container = { repeating: false, dataModelGroup: null } as ICreateFormContainer;
-        yield call(FormDesignerActionDispatchers.addFormContainerFulfilled, container, activeContainer);
-      }
+    if (!containerId) {
+      // if not containerId set it to base-container
+      containerId = Object.keys(formDesignerState.layout.order)[0];
+    }
+    if (!position) {
+      // if position is undefined, put it on top
+      position = 0;
     }
-
     yield call(
       FormDesignerActionDispatchers.addFormComponentFulfilled,
       component,
       id,
       position,
-      activeContainer,
+      containerId,
       callback,
     );
     return id; // returns created id
   } catch (err) {
+    console.error(err);
```

Comment:
should we console.log errors?

Reply:
Copy-paste comment:
console.error writes the red text in the console. Log is for logging, error is for error. We do have this errorhandling component though. Can change this to send a rejectedAction and add the error to the error-component.